### DISCUSSION TITLE
Update problem

### DISCUSSION THREAD

**Discussion by KrystianPlichta** [netbox-community/netbox-docker#564] [Getting Help] [answered]
Hi, im trying to update my netbox ver.2.10.8 to the latest version but i cant directly, i have to update to version 2.11.12 an then to the latest.

I have error when i trying to pull

This is my /var/netbox/netbox-docker/docker-compose.override.yml file

version: '3.4'
services:
  netbox:
    image: netboxcommunity/netbox:2.11.12
    ports:
      - 8000:8080

**Comment by tobiasge** [comment, marked as answer]
You a typo in the tag. It should be `v2.11.12`.

**Reply by KrystianPlichta** [reply]
Thank you ;)

**Comment by KrystianPlichta** [comment]
On the latest version, import option dont work, any idea ?
